### PR TITLE
Update AWS Java SDK to 1.11.211

### PR DESCRIPTION
**Pull request by Vlatombe**
Add support for x1e.32xlarge instance type

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <relativePath />
   </parent>
   <artifactId>aws-java-sdk</artifactId>
-  <version>1.11.120-SNAPSHOT</version>
+  <version>1.11.211-SNAPSHOT</version>
```
Comment (johnou) [inline pom.xml]: why increment this version? shouldn't it be part of the maven release process?

Comment (Vlatombe) [reply pom.xml]: Doesn't really matter. This is mostly to get the right default value when doing the maven release later on.

**Comment by Vlatombe** [comment]
Done https://github.com/jenkinsci/aws-java-sdk-plugin/commit/5cab28b8f93d0a69ff9f2ce2cd4e382b61bf9cad

**Comment by ilchemla** [comment]
Thanks a lot! @Vlatombe

**Pull request closed without merging**